src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +36,11 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect
+              ? isAnon
+                ? COLORS.anon_primary + backgroundOpacity15PercentHexCode
+                : COLORS.signed_primary + backgroundOpacity15PercentHexCode
+              : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color logic has been updated to handle the case where the user is anonymous. This looks fine, but it might be more readable if you extract this logic into a separate function or variable.
```diff
- backgroundColor: isSelect
-   ? isAnon
-     ? COLORS.anon_primary + backgroundOpacity15PercentHexCode
-     : COLORS.signed_primary + backgroundOpacity15PercentHexCode
-   : COLORS.white
+ const getBackgroundColor = () => {
+   if (!isSelect) return COLORS.white;
+   return isAnon ? COLORS.anon_primary + backgroundOpacity15PercentHexCode : COLORS.signed_primary + backgroundOpacity15PercentHexCode;
+ };
+ ...
+ backgroundColor: getBackgroundColor()
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Ok will do